client/package.json

```diff
@@ -79,6 +79,7 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.26.1",
     "husky": "^7.0.2",
+    "jest": "^26.6.0",
```

Comment:
Since this is the first set of tests that we have, can you also add the workflow for running the tests as well?